Assignment 3/main.cpp

```diff
@@ -0,0 +1,245 @@
+#include <iostream>
+#include <fstream>
+#include <limits>
+#include "./reader.h"
+
+
+using std::cout;
+using std::cin;
+using std::string;
+using std::endl;
+
+
+void Initialization() {
+    string Username = "";
+    do {
+        cout << "Enter your name(first and last)" << endl;
+        getline(cin, Username);
+    } while (Username == "");
+
```

Comment:
Good use of making inti and edit functions but if you just call the config file to the library program it will save you alot of lines.